tools/Common/IceRpc.Telemetry.Internal/Program.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc;
+using IceRpc.Retry;
+using IceRpc.Telemetry.Internal;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+const int timeout = 3000; // The timeout for the RPC call in milliseconds.
+const int maxAttempts = 3; // The maximum number of attempts to retry the RPC call.
+const string uri = "icerpc://localhost"; // The URI of the server.
```

Comment:
Tried to just pick reasonable numbers. Fine with changing these values.

Reply:
How do retry work here? The only operation that we call is oneway but non-idempotent.